src/main/java/Duke.java

```diff
@@ -1,10 +1,77 @@
+package main.java;
+
+import java.util.*;
 public class Duke {
     public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you");
+        String inp = in.nextLine();
```

Comment:
Single comment for whole file:
- Consider not shortening variable names, especially if its already short enough < 8 chars
- `inp`, `comm`, `tas` requires context to understand so it may be better to leave them as is or name differently